uk.co.compucorp.civicrm.hrleaveandabsences/xml/Menu/import.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0"?>
+<menu>
+  <item>
+    <path>civicrm/admin/leaveandabsences/import</path>
+    <page_callback>CRM_HRLeaveAndAbsences_Import_Controller</page_callback>
+    <title>Import Leave/Absence Requests</title>
+    <access_arguments>administer leave and absences</access_arguments>
+  </item>
+</menu>
```

Comment:
@tunbola about the xml file: yes, please add it directly to `xml/Menu/hrleaveandabsences.xml`. Unless there's a technical reason, there's no point in having a separate file just for that.

about the menu: sorry, I missed that. Unfortunately, adding it to the hook is not enough. Since the extension has now been released (as a beta, to compucorp) we need to start using upgraders for this kind of stuff. Otherwise the menu won't be available on existing sites